var/spack/repos/builtin/packages/gconf/package.py

```diff
@@ -14,13 +14,10 @@ class Gconf(AutotoolsPackage):
 
     version('3.2.6', sha256='1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c')
 
+    depends_on('pkgconfig', type='build')
     depends_on('glib@2.14.0:')
     depends_on('libxml2')
-
-    # TODO: add missing dependencies
-    # gio-2.0 >= 2.31.0
-    # gthread-2.0
-    # gmodule-2.0 >= 2.7.0
-    # gobject-2.0 >= 2.7.0
-    # dbus-1 >= 1.0.0
-    # dbus-glib-1 >= 0.74
+    depends_on('dbus')
+    depends_on('dbus-glib')
+    depends_on('orbit2')
+    depends_on('perl-xml-parser')
```

Comment:
Is this type=build or type=build/run?

Reply:
Considering functions, all new packages seems to be "type=build/run".

Reply:
The default is build/link. I don't think this Perl package even installs a library, right?

Reply:
I was confused. ”build/link" seems enough in this case.

Reply:
But we don't do any linking, right? In that case, it should be `type='build'`

Reply:
I checked installed files again.
Path to `perl` package is included in `gconf` installed files such as `bin/gconftool-2`.
So when gconf is run, perl and xml-parser will be called.
I think type = 'build/run' is needed for perl-xml-parser, others are default.

Reply:
Updated patch